src/resources/i18n/translations/en.json

```diff
@@ -742,26 +742,26 @@
   "Yes enable tracking": "Yes, track me",
   "Yes save them": "Yes, save them",
   "Your premium membership is still valid": "Your premium membership is still valid. If you would ",
-  "You are about to be navigated to a website outside the app Are you sure you want to leave brandName": "Leave Podverse for a website outside the app?",
+  "You are about to be navigated to a website outside the app Are you sure you want to leave brandName": "Open website in web browser?",
```

Comment:
I was of the impression that we have to be explicit about when a user is "Leaving Podverse" for app store compliance. I'll merge this though and we'll see if they flag us.

Reply:
When I click the GitHub link in "About" I am not actually leaving Podverse.
It just opens a web browser.